src/App.tsx

```diff
@@ -1,14 +1,76 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import * as api from './api'
+
+
+export enum SortType {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
+}
+
+
+const getFilteredTodos = (
```

Comment:
There is a typo in the function name `getFilteredTodods`. It should be corrected to `getFilteredTodos` to prevent potential runtime errors.